connectivity/libraries/nanostack-libservice/test/libService/unittest/nsnvmhelper/main.cpp

```diff
@@ -1,5 +1,18 @@
-/*
- * Copyright (c) 2015 ARM Limited. All rights reserved.
+/* mbed Microcontroller Library
+ * Copyright (c) 2015 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
```

Comment:
```suggestion
```
We prefer the short header (only Copyright + SPDX-License-Identifier) now? Could you confirm this? @ARMmbed/mbed-os-maintainers

Reply:
the license guide as it was approved by the legal says both should be in the file.

> Begin each source header with your copyright line, the SPDX identifier and the standard header for the license that applies to that single file, if it has one. (See SPDX Specification, Appendix V.)

Although some of the newer files have only SDPX and copyright and were approved. I'll review with @harmut01.

I would leave this as it is as it follows the current guideline and only updates the files that are already in the tree.